operations/research-and-development/product/release-process/focalboard-release.md

```diff
@@ -0,0 +1,48 @@
+# Focalboard Release Process
```

Comment:
@chenilim 
 - I created a basic outline of the Focalboard release process below, please review for any suggested edits. The guide can be expanded on in the future as we work on releases.
 - Also, wondering if we'll be using IC Playbook for releases in the future https://community-release.mattermost.com/core/pl/9etgsq75sf8u7nf9g5e7smjqte.